src/ui/table/additional/AdditionalInfoWidget.cpp

```diff
@@ -96,10 +98,25 @@ AdditionalInfoWidget::calculateWidth()
 }
 
 
+bool
+AdditionalInfoWidget::isValidEdit()
+{
+    const auto mainInfoText = getMainInfoText(); 
+    if (!mainInfoText.isEmpty() && mainInfoTextCache != mainInfoText) {
+        return true;
+    }
+    return false;
+}
+
+
 bool
 AdditionalInfoWidget::eventFilter(QObject* object, QEvent* event)
 {
     if (object == m_additionalInfoLineEdit && event->type() == QEvent::FocusIn) {
+        const auto mainInfoText = getMainInfoText();
+        if (mainInfoTextCache.isNull()) {
```

Comment:
It makes sense to not rely on an empty string, but using the string's adress can be dangerous because of possible nullptr exceptions. 
Better to initialize just an empty string and use a bool member which is false per default and set it to `true` if the condition is true. Imo this is safer than using a string's address.